docs/.ci/generate-docs.sh

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
```

Comment:
I'm not sure if this script should be moved to `kyma` repository. It is strictly related to `website` repository and this is a generation of documentation for `website` purposes, not `kyma`. In that case it should stay in `website`.

Reply:
Sorry, @akucharska for my previous comment about putting the scripts in separate directory. I should have noticed, that they don't fit into the `kyma` repository. @michal-hudy is right, they should be still located on `website` repo.